stable/azuredisk-csi-driver/values.yaml

```diff
@@ -4,31 +4,31 @@ image:
     tag: v0.5.0
     pullPolicy: Always
   csiProvisioner:
-    repository: mcr.microsoft.com/oss/kubernetes-csi/csi-provisioner
+    repository: quay.io/k8scsi/csi-provisioner
     tag: v1.4.0
```

Comment:
v1.5.0 got released, but maybe something for the next change

Reply:
look at https://github.com/kubernetes-sigs/azuredisk-csi-driver for the next upgrade